myhpi/static/js/navbar.js

```diff
@@ -231,8 +231,8 @@ const moveRootLevelToDesktopLayout = () => {
     const rootNavLevelContainer = document.querySelector("#nav-level-0")
     const rootLevel = document.querySelector("#nav-item-container-root")
     rootNavLevelContainer.appendChild(rootLevel)
+    document.getElementById("user-information-username").hidden = false
```

Comment:
This breaks the website if no user is logged in, as no span with this ID exists in this situation.  
This could be fixed in JS, but I noticed that the use of the `hidden` attribute is [against the specs](https://stackoverflow.com/a/6708403/8325653).

So I'd opt for a CSS-only solution: Simply add the respective [Boostrap built-in class](https://getbootstrap.com/docs/5.0/utilities/display/) (with the appropriate breakpoint) to the span